Lib/test/test_dis.py

```diff
@@ -1041,11 +1041,6 @@ def get_disassembly(self, func, lasti=-1, wrapper=True, **kwargs):
         return output.getvalue()
 
 
-if sys.flags.optimize:
-    code_info_consts = "0: None"
-else:
-    code_info_consts = "0: 'Formatted details of methods, functions, or code.'"
-
```

Comment:
Failing tests were:

<img width="317" alt="image" src="https://user-images.githubusercontent.com/1750152/170062729-b9d4a408-0121-4c17-957b-2fbd20eccffb.png">

It seemed as though this condition was not necessary anymore, though I'm unsure if it is the expected behaviour.